docs/theory/apportion_abundance.md

```diff
@@ -0,0 +1,65 @@
+(apportion-abundance)=
+# Apportioning kriged abundance
+
+{{<b>Perhaps a link to the [biomass apportionment page](../theory/apportion_biomass.md) or additional (brief) references/re-explanations here?</b>}}
+
+Summed biomass estimates along $\ell$ and $\alpha$ for each $s$ along transect interval $k$ for aged ($B_{s, \ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{s, \ell}^{k,\textrm{unaged}}$) kriged biomass estimates are summed together ($\hat{B}_{s}^{k}$) by first summing $B_{s, \ell, \alpha}^{k,\textrm{aged}}$ across $\alpha$ for each $\ell$ and then adding that quantity with $B_{s, \ell}^{k,\textrm{unaged}}$. The result is then summed across all $\ell$ to yield $\hat{B}_{s}^{k}$. This is done in tandem with the unsexed aged ($B_{\ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{\ell}^{k,\textrm{unaged}}$) kriged biomass estimates to yield the total biomass ($\hat{B}^{k}$). This not calculated via adding male ($\hat{B}_{M}^{k}$) and female ($\hat{B}_{F}^{k}$) since $\hat{B}^{k}$ is inclusive of male, female, <i>and</i> unsexed fish.
```

Comment:
I wonder if the part about 
> This is done in tandem with the unsexed aged ($B_{\ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{\ell}^{k,\textrm{unaged}}$) kriged biomass estimates
Should be 
> This is done in tandem with the unsexed aged ($B_{\textrm{unsexed}, \ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{\textrm{unsexed}\ell}^{k,\textrm{unaged}}$) kriged biomass estimates

Since in all expression so far when we omit subscript index it means it is summed over all partition of that index (e.g., omitting $\alpha$ means summing over all ages)

Reply:
Ah, since you've pointed that out, I actually think my original language was too ambiguous. By "unsexed aged ($B_{\ell, \alpha}^{k, \textrm{ aged }}$) [...]", I actually meant to refer to this as the _total_ biomass which is inclusive of male, female, and unsexed fish. There is a different section of the original Matlab code (albeit for the transect analysis and not the kriging) where the unsexed biomass is intentionally reported separately from sexed (or male/female). Similar to other aspects we've discussed for the kriging apportionment, I don't see why that couldn't be extended here. Putting that aside, I think $B_{\textrm{ unsexed}, \ell, \alpha}^{k, \textrm{ aged } }$ could create some confusion with biomass estimates for specifically unsexed fish I think, something like $B_{\textrm{all}, \ell, \alpha}^{k, \textrm{aged}}$ would be better? Or maybe  $B_{\textrm{total}, \ell, \alpha}^{k, \textrm{aged}}$ since we use $\textrm{all}$ to denote both aged and unaged?